mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulProfileScreen.kt

```diff
@@ -27,16 +49,28 @@ class FetchedUserProfileScreenState(
  *
  * @param uid of the player.
  * @param modifier the [Modifier] for this composable.
+ * @param contentPadding the [PaddingValues] to apply to this screen.
  */
 @Composable
 fun StatefulProfileScreen(
     uid: String,
     modifier: Modifier = Modifier,
+    contentPadding: PaddingValues = PaddingValues(),
 ) {
   val socialFacade = LocalSocialFacade.current
-  val profile by remember(socialFacade, uid) { socialFacade.profile(uid) }.collectAsState(null)
-  val state = profile?.let { FetchedUserProfileScreenState(it) }
-  if (state != null) {
-    ProfileScreen(state, modifier)
-  }
+  val chessFacade = LocalChessFacade.current
+  val emptyProfile: Profile =
+      object : Profile {
+        override val emoji: String = ""
+        override val name: String = ""
+        override val backgroundColor: Profile.Color = Profile.Color.Default
+        override val uid: String = ""
+        override val followed: Boolean = false
+      }
+  val fetchedProfile by
+      remember(socialFacade, uid) { socialFacade.profile(uid) }.collectAsState(null)
+  val profile = fetchedProfile ?: emptyProfile
+  val scope = rememberCoroutineScope()
+  val state = remember(profile) { FetchedUserProfileScreenState(profile, chessFacade, scope) }
```

Comment:
Bug : you must specify some dependencies. If the `ChessFacade` or the `CoroutineScope` were to be changed, you'd need to re-create the `FetchedUserProfileScreenState`.

```suggestion
  val state = remember(profile, chessFacade, scope) { FetchedUserProfileScreenState(profile, chessFacade, scope) }
```